octoprint_mrbeam/templates/conversion_dialog.jinja2

```diff
@@ -193,7 +193,7 @@
                     <p>
                         <i class="icon-warning-sign" style="color:red;"
                            data-bind="visible: material_safety_notes()"></i>
-                        <span id="safety_notes" class="" data-bind="text: material_safety_notes"></span>
+                        <span id="safety_notes" class="" data-bind="text: material_safety_notes, css:{ incompatible_safety_warning: $root.selected_material_compatibility() === false }"></span>
```

Comment:
personaly prefer `!something` rather then `something === false` - but perhaps it is because of `null` behaviour?

Reply:
resolved